packages/google_workspace/data_stream/rules/fields/ecs.yml

```
@@ -0,0 +1,70 @@
+- external: ecs
+  name: ecs.version
+- external: ecs
+  name: event.action
+- external: ecs
+  name: event.created
+- external: ecs
+  name: event.id
+- external: ecs
+  name: event.ingested
+- external: ecs
+  name: event.kind
+- external: ecs
+  name: event.original
+- external: ecs
+  name: event.provider
+- external: ecs
+  name: organization.id
+- external: ecs
+  name: related.hosts
+- external: ecs
+  name: related.ip
+- external: ecs
+  name: related.user
+- external: ecs
+  name: rule.id
+- external: ecs
+  name: rule.name
+- external: ecs
+  name: source.address
+- external: ecs
+  name: source.as.number
+- external: ecs
+  name: source.as.organization.name
+- external: ecs
+  name: source.geo.city_name
+- external: ecs
+  name: source.geo.continent_name
+- external: ecs
+  name: source.geo.country_iso_code
+- external: ecs
+  name: source.geo.country_name
+- name: source.geo.location
+  level: core
+  type: geo_point
+  description: Longitude and latitude.
```

```suggestion
- external: ecs
  name: source.geo.location
```
https://www.elastic.co/guide/en/ecs/current/ecs-geo.html#field-geo-location